README.md

```diff
@@ -106,6 +106,23 @@ adding an abstraction layer a-la-Firewall. It's rather limited how far you can
 go so if you need rather complex rules or you can speak nftables it's
 recommended to use `nftables::rule` directly.
 
+## Facts
+
+One structured fact `nftables` is available
```

Comment:
Maybe it'd be nice to add an explanation for each key in the hash.

Reply:
Added something.